qadence/noise/protocols.py

```diff
@@ -149,15 +149,18 @@ def list(cls) -> list:
         return list(filter(lambda el: not el.startswith("__"), dir(cls)))
 
     def filter(self, protocol: NoiseEnum | str) -> NoiseHandler | None:
-        is_protocol: list = [p == protocol or isinstance(p, protocol) for p in self.protocol]  # type: ignore[arg-type]
-        return (
-            NoiseHandler(
+        is_protocol: list = list()
```

Comment:
`is_<smtg>` generally conveys the idea of a predicate. Here, I'd use `protocols` instead.

Reply:
Updated with `protocol_matches`